backend-rails/spec/requests/auth_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
```

Comment:
Please delete or update this spec

Reply:
Done here: [208b318](https://github.com/espoo-dev/group-habits/pull/441/commits/208b318fd05f5eb897dd49f380812f22a5ba592e)